<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandsEditor</class>
 <widget class="XUPPageEditor" name="CommandsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commands</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/xupeditor/icons/xupeditor/compiler.png</normaloff>:/xupeditor/icons/xupeditor/compiler.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbCommands">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QToolButton" name="tbCommandAdd">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/tools/icons/tools/add.png</normaloff>:/tools/icons/tools/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="tbCommandUp">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/tools/icons/tools/up.png</normaloff>:/tools/icons/tools/up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="tbCommandDown">
        <property name="text">
         <string>Down</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/tools/icons/tools/down.png</normaloff>:/tools/icons/tools/down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QFrame" name="fEditor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="lCommandText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Text :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="7">
          <widget class="Line" name="lCommand">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lCommandParsers">
           <property name="text">
            <string>Parsers :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="leCommandText"/>
         </item>
         <item row="1" column="3" rowspan="5">
          <widget class="QListWidget" name="lwCommandParsers">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="lCommandCommand">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Command :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="leCommandCommand"/>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="tbCommandCommand">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/tools/icons/tools/file.png</normaloff>:/tools/icons/tools/file.png</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="lCommandWorkingDirectory">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Working Directory :</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="leCommandWorkingDirectory"/>
         </item>
         <item row="5" column="1">
          <widget class="QToolButton" name="tbCommandWorkingDirectory">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/tools/icons/tools/icon.png</normaloff>:/tools/icons/tools/icon.png</iconset>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="cbCommandSkipOnError">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Skip On Error</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QCheckBox" name="cbCommandTryAll">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Try All</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="5">
       <widget class="QTreeView" name="tvCommands">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="tbHelp">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/tools/icons/tools/help.png</normaloff>:/tools/icons/tools/help.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XUPPageEditor</class>
   <extends>QFrame</extends>
   <header>xupmanager/gui/XUPPageEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbCommandTryAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>lwCommandParsers</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
